components/HeroLanding/HeroButton.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { CustomLink } from 'components';
+import { Box, Button, Icon } from 'ui-kit';
+import { gtag } from 'lib/analytics';
+import { useCurrentBreakpoint } from 'hooks';
+import Styled from './HeroLanding.styles';
+
+const HeroButton = ({ actions, iconNames, ...props }) => {
+  const currentBreakpoint = useCurrentBreakpoint();
+  let marginLeft;
+  if (actions.length === 2 && currentBreakpoint.isSmall) {
+    marginLeft = '0';
+  } else if (actions.length === 2 && !currentBreakpoint.isSmall) {
+    marginLeft = 'l';
+  }
+  return (
+    <Styled.Buttons>
+      {actions.map((action, i) => (
+        <CustomLink
+          key={i}
+          size="l"
+          Component={Button}
+          onClick={() => [
+            gtag.trackEvent({
+              category: `Hero Landing - ${props?.heroTitle}`,
+              label: `${action.title} - Button`,
+              action: action.url,
+            }),
+          ]}
+          width={{ _: '100%', md: 'auto' }}
+          href={action.url}
+          my={{ _: 'xs', md: 'base' }}
+          mr={{ _: '0', md: i > 1 ? '' : 's' }}
+          ml={marginLeft}
+          backgroundColor="rgba(0, 114, 188, 0.40)"
+          borderRadius="100px"
+          py="s"
+          px="base"
+          lineHeight="1"
+          display="flex"
+          flexDirection="row"
+          {...action}
+        >
+          <Box>
+            {action.greenTitle && (
```

Comment:
What if we called this `subtitle` and the other subtitle `description`? 🤔 This way if we ever wanted to change the color down the road we won't have to rename any properties.